exo-create/main_test.go

```diff
@@ -0,0 +1,162 @@
+	cmd.Dir = appDir
+	in, err = cmd.StdinPipe()
+	cmd.Stdout = &out
+	if err != nil {
+		return err
+	}
+	if err = cmd.Start(); err != nil {
+		return fmt.Errorf("Error running %s\nError:%s", command, err)
+	}
+	return nil
+}
+
+func validateTextContains(haystack, needle string) error {
+	if strings.Contains(haystack, needle) {
+		return nil
+	}
+	return fmt.Errorf("Expected:\n\n%s\n\nto include\n\n%s", haystack, needle)
+}
+
+func enterInput(row *gherkin.TableRow) error {
+	_, input := row.Cells[0].Value, row.Cells[1].Value
+	_, err := in.Write([]byte(input + "\n"))
+	return err
+}
+
+// nolint gocyclo
+func FeatureContext(s *godog.Suite) {
+	var cwd, childOutput string
+	var err error
+
+	s.BeforeSuite(func() {
+		var err error
+		cwd, err = os.Getwd()
+		if err != nil {
+			panic(err)
+		}
+	})
+
+	s.Step(`^executing "([^"]*)"$`, func(command string) error {
+		return run(cwd, command)
+	})
+
+	s.Step(`^starting "([^"]*)" in the terminal$`, func(command string) error {
+		appDir = path.Join(cwd, "tmp")
+		if err := testHelpers.EmptyDir(appDir); err != nil {
+			return err
+		}
+		return run(cwd, command)
+	})
+
+	s.Step(`^entering into the wizard:$`, func(table *gherkin.DataTable) error {
+		for _, row := range table.Rows[1:] {
+			if err := enterInput(row); err != nil {
+				return err
+			}
+		}
+		defer in.Close()
+		return nil
+	})
+
+	s.Step(`^running "([^"]*)" in the terminal$`, func(command string) error {
+		appDir = path.Join(cwd, "tmp")
+		if err := testHelpers.EmptyDir(appDir); err != nil {
+			return err
+		}
+		return run(cwd, command)
+	})
+
+	s.Step(`^waiting until I see "([^"]*)" in the terminal$`, func(expectedText string) error {
+		if err := cmd.Wait(); err != nil {
+			return err
+		}
+		childOutput = out.String()
+		return validateTextContains(childOutput, expectedText)
+	})
+
+	s.Step(`^it prints "([^"]*)" in the terminal$`, func(expectedText string) error {
+		if err = cmd.Wait(); err != nil {
+			return err
+		}
+		childOutput = out.String()
+		return validateTextContains(childOutput, expectedText)
+	})
+
+	s.Step(`^my application contains the file "([^"]*)" with the content:$`, func(filePath string, expectedContent *gherkin.DocString) error {
+		bytes, err := ioutil.ReadFile(path.Join(appDir, filePath))
+		if err != nil {
+			return err
+		}
+		return validateTextContains(strings.TrimSpace(string(bytes)), strings.TrimSpace(expectedContent.Content))
+	})
+
+	s.Step(`^my workspace contains the file "([^"]*)" with content:$`, func(fileName string, expectedContent *gherkin.DocString) error {
+		bytes, err := ioutil.ReadFile(path.Join(appDir, fileName))
+		if err != nil {
+			return err
+		}
+		return validateTextContains(strings.TrimSpace(string(bytes)), strings.TrimSpace(expectedContent.Content))
+	})
+
+	s.Step(`^my workspace contains the empty directory "([^"]*)"`, func(directory string) error {
+		if _, err := os.Stat(path.Join(appDir, directory)); err == nil {
+			return nil
+		}
+		if os.IsNotExist(err) {
```

Comment:
You might want to verify that the file system entry is actually a directory here.